src/patterns/question/example-question-anatomy.njk

```diff
@@ -61,6 +61,22 @@ noTabs: true
             </th>
         </tr>
     </tbody>
+    <tbody class="ons-pl-question-anatomy__row">
+        <tr>
+            <td class="ons-pl-question-anatomy__component">
+                {% call onsPanel({
+                    "type": "warn"
+                }) %}
+                    <p>This question is important</p>
```

Comment:
To avoid users thinking you can use this just to mark a question as 'important' or 'mandatory', I would just change this to an obvious placeholder, or change the whole example used to match a real use case where all elements would be used.
```suggestion
                    <p>This is a warning notice</p>
```